File: Bars.GkhGJI/libs/B4/model/Dict/KindStatementGJI.js
Ext.define('B4.model.dict.KindStatementGji', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'KindStatementGji'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'Name' },
        { name: 'Code' },
        { name: 'Description' },
        { name: 'Postfix' }
    ]
});

File: Bars.GkhGJI/Entities/Dict/KindStatementGji.cs
namespace Bars.GkhGji.Entities
{
    using Bars.Gkh.Entities;

    /// <summary>
    /// Вид обращения
    /// </summary>
    public class KindStatementGji : BaseGkhEntity
    {
        /// <summary>
        /// Наименование
        /// </summary>
        public virtual string Name { get; set; }

        /// <summary>
        /// Код
        /// </summary>
        public virtual string Code { get; set; }

        /// <summary>
        /// Описание
        /// </summary>
        public virtual string Description { get; set; }

        /// <summary>
        /// Постфикс для формирования номера обращения
        /// </summary>
        public virtual string Postfix { get; set; }
    }
}